app/services/idv/steps/send_link_step.rb

```diff
@@ -3,11 +3,15 @@ module Steps
     class SendLinkStep < DocAuthBaseStep
       def call
         capture_doc = CaptureDoc::CreateRequest.call(current_user.id)
-        SmsDocAuthLinkJob.perform_now(
-          phone: permit(:phone),
-          link: link(capture_doc.request_token),
-          app: 'login.gov',
-        )
+        begin
+          SmsDocAuthLinkJob.perform_now(
+            phone: permit(:phone),
+            link: link(capture_doc.request_token),
+            app: 'login.gov',
+          )
+        rescue Twilio::REST::RestError
```

Comment:
I think you may want to rescue from the Verify error the jobs can raise as well: https://github.com/18F/identity-idp/blob/master/app/controllers/users/two_factor_authentication_controller.rb#L10